src/Sulu/Bundle/ContentBundle/Resources/public/js/components/teaser-selection/main.js

```diff
@@ -213,11 +213,16 @@ define([
                                     return item['type'] === type.name;
                                 }),
                                 selectCallback: function(item) {
-                                    data.push(item);
+                                    var filteredArray = _.filter(data, function (arrayItem) {
+                                        return arrayItem.id === item.id && arrayItem.type === item.type;
+                                    });
+
+                                    if (0 === filteredArray.length) {
+                                        data.push(item);
+                                    } else {
+                                        data = _.without(data, _.findWhere(data, item));
+                                    }
                                 },
-                                deselectCallback: function(item) {
```

Comment:
I'm little bit confused why this function was removed or why it did exist 🙈

Reply:
Just tested it, and it seems to work this way... Looks to me like the deselectCallback is never called :see_no_evil: Therefore this change is fine for me.